uaa-overview.html.md.erb

```diff
@@ -9,9 +9,7 @@ UAA provides enterprise-scale identity management features. For example, it is u
 
 * [Single Sign-On for VMware Tanzu](https://pivotal.io/platform/services/identity-and-security/sso): Identity services for apps and APIs running on VMware Tanzu Application Service (TAS for VMs)
 
-* [GE Predix Identity Services](https://docs.predix.io/en-US/content/service/security/user_account_and_authentication/uaa-service-overview)
```

Comment:
I Think yes, because here in certified / or only partners I cannot found GE information
https://www.cloudfoundry.org/the-foundry/

btw. IBM deprecated all CF services and end also mid of year all CF services

Reply:
but if you have link we can use this. The current one https://docs.predix.io/en-US/content/service/security/user_account_and_authentication/uaa-service-overview is simply redirected to a generic page